app/objects/c_ability.py

```diff
@@ -46,7 +47,8 @@ def build_ability(self, data, **_):
 class Ability(FirstClassObjectInterface, BaseObject):
 
     schema = AbilitySchema()
-    display_schema = AbilitySchema(exclude=['repeatable', 'language', 'code', 'build_target'])  # may need to fix for id=self.unique
+    # may need to fix for id=self.unique
```

Comment:
is this resolved?

Reply:
I have no idea. I didn't create that line, I just moved the comment after extending the line slightly due to changes so that it would be in line with our style.